common/changes/office-ui-fabric-react/thomabr-MoveSharedCellLogicToGrid_2017-08-03-16-49.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "This change moves a majority of the logic away from the SwatchColorPickerOption class into the GridCell class. The Grid class will handle most of the navigation/selection logic and only push an onRenderItem callback to the consumers where they can specify how the cell should be rendered.",
```

Comment:
Agreed. On top of no one using it yet, this should only be adding to the API, not changing it in a breaking way